pages/sign-in/kakao/index.page.tsx

```diff
@@ -75,5 +85,9 @@ export default function Kakao({ id, isComplete }: KakaoProps) {
     router.push(PAGE_PATH.DASHBOARD);
   }, [isComplete]);
 
-  return <div>로그인 요청중입니다.</div>;
+  return (
+    <div className={cn("container")}>
+      <Lottie animationData={loadingLottie} loop play className={cn("loading-lottie")} />
```

Comment:
```suggestion
      <Lottie animationData="/images/login-loading.json" loop play className={cn("loading-lottie")} />
```
P5: 만약에 이걸 사용한다고 했을 때 이렇게 직접 경로는 못넣나요??

Reply:
네ㅠㅠ 이미지처럼 직접 경로에 넣었더니 파일을 못읽더라구요.... 그래서 어쩔 수 없이 import 시켜서 넣어주었습니다ㅠㅠ